micro_dl/input/dataset.py

```diff
@@ -305,21 +308,21 @@ def _augment_image(self,
             trans_image = np.rot90(
                 input_image,
                 k=1,
-                axes=(0 + add_dim, 1 + add_dim),
+                axes=(-2, -1),
```

Comment:
Good catch. I changed this because previously it was rotating over channel dimension when input was 4d (as in 2d unet), which caused an error. 

To simplify code, PyTorch implementation only accepts data in format (...zyx or ...zxy). If we want to handle other formats I think the best way is to convert any other formats ((b,z,y,x,c) for example) to (b,c,z,y,x) on read-in and write.